src/main/java/mezz/jei/autocrafting/RecipeBookmarkItem.java

```diff
@@ -51,6 +51,18 @@ public RecipeBookmarkItem(List<I> aliases, int amount) {
         this.amount = amount;
     }
 
+    public RecipeBookmarkItem(RecipeBookmarkItem<I> other) {
+        super(other.ingredient);
+        this.aliases = other.aliases;
```

Comment:
Since aliases are copied over too, if `foundAliases` was true, shouldn't the copied object also set it to true anyway?